test/e2e/csv_e2e_test.go

```diff
@@ -36,10 +36,10 @@ import (
 var _ = Describe("CSV", func() {
 	It("create with unmet requirements mini kube version", func() {
 
-		defer cleaner.NotifyTestComplete(GinkgoT(), true)
+		defer cleaner.NotifyTestComplete(true)
```

Comment:
Shouldn't we use AfterEach?

Reply:
@awgreene , these will be changed when I pick up csv_e2e_test.go for ginkgo conversion. For now, this PR is only concerned with dynamic_e2e_test and some of the changes in helper methods